.github/workflows/release.yml

```diff
@@ -104,10 +97,8 @@ jobs:
       - name: Create Dmg Installer
         run: ./gradlew buildDmgPackage
```

Comment:
We should pass the `version` to this command as well, see https://stackoverflow.com/a/40239092